types/nodemailer-sendgrid/nodemailer-sendgrid-tests.ts

```diff
@@ -0,0 +1,17 @@
+import sendgrid from 'nodemailer-sendgrid';
+import nodemailer from 'nodemailer';
+
+const opts: sendgrid.SendgridOptions = { apiKey: 'XXXXXXXXXXXXXXXX' };
+const transport: nodemailer.Transport = sendgrid(opts);
+const mailOptions: nodemailer.SendMailOptions = {
+    from: 'Foo Bar ✔ <foo@bar.com>',
+    to: 'foo@baz.com, foo@baz.com',
+    subject: 'Hello world ✔',
+    text: 'Hello world ✔',
+    html: '<b>Hello world ✔</b>'
+};
+
+transport.send(mailOptions, (error: Error, info: nodemailer.SentMessageInfo): void => {
+	// nothing
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/nodemailer-sendgrid/nodemailer-sendgrid-tests.ts;
  Line: 15;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.